src/components/dropdown/__tests__/index.js

```diff
@@ -0,0 +1,105 @@
+const ReactDOM = require('react-dom');
+//import user from 'focus-core/user';
+import Dropdown from '../';
+
+const {renderIntoDocument,findRenderedDOMComponentWithClass} = TestUtils;
+
+const alertSpy = sinon.spy();
+
+const actions = [
+    {label: "Action_a", msg: "Action a"},
+    {label: "Action_b", msg: "Action b"},
+    {label: "Action_c", msg: "Action c"},
+    {label: "Action_d", msg: "Action d"},
+];
+
+const operationList = [
+    {
+        label: actions[0].label, action: function () {
+        const alertObj = {msg: actions[0].msg};
+        alertSpy(alertObj);
+    }, style: "class"
+    },
+    {
+        label: actions[1].label, action: function () {
+        const alertObj = {msg: actions[1].msg};
+        alertSpy(alertObj);
+    }
+    },
+    {
+        label: actions[2].label, action: function () {
+        const alertObj = {msg: actions[2].msg};
+        alertSpy(alertObj);
+    }
+    },
+    {
+        label: actions[3].label, action: function () {
+        const alertObj = {msg: actions[3].msg};
+        alertSpy(alertObj);
+    }
+
+    }
+];
+
+class TestWrapper extends React.Component {
+    render() {
+        const {children,mockedValue, ...otherProps} = this.props;
+        return (
+            <div className={mockedValue} ref={mockedValue}>{children}</div>
```

Comment:
What is the interest of `className` is it `TestUtils.findComponentWithClassName`